optimum/intel/utils/modeling_utils.py

```diff
@@ -110,3 +110,25 @@ def _find_files_matching_pattern(
         files = [Path(p) for p in repo_files if re.match(pattern, str(p)) and str(p.parent) == subfolder]
 
     return files
+
+
+def replace_customized_linear_with_linear(model):
```

Comment:
could you add type hints + docstring for all ?

Reply:
Done.